src/ucp/core/ucp_listener.c

```diff
@@ -77,6 +77,12 @@ ucs_status_t ucp_conn_request_query(ucp_conn_request_h conn_request,
         }
     }
 
+    if (attr->field_mask & UCP_CONN_REQUEST_ATTR_FIELD_WORKER_ID) {
+        /* coverity[overrun-buffer-val] */
+        attr->worker_id = ucp_address_get_uuid(
```

Comment:
wire compatible - how to handle connection requests from older UCX version?
need to add uuid flag to addr_mode